service-api/module/Application/src/Controller/IdentityController.php

```diff
@@ -187,4 +187,91 @@ public function validatePassportAction(): JsonModel
 
         return new JsonModel($response);
     }
+
+    public function getKbvQuestionsAction(): JsonModel
+    {
+        $uuid = $this->params()->fromRoute('uuid');
+
+        /**
+         * @psalm-suppress PossiblyUndefinedVariable
+         */
+        $response[$uuid] = [
+            "one" => [
+                "id" => 1,
+                "question" => "Who provides your mortgage?",
+                "number" => "one",
```

Comment:
It looks like this is just used to identify the question in the confirmation request? Probably easier to just use the numerals (i.e. `id`) rather than having two fields describing the same thing

Reply:
removed the ids instead